_data/changelogs/docs-settings.yml

```
@@ -2,6 +2,17 @@ title: Settings
 type: utility
 changelogURL:
 items:
+  - date: 2023-05-12
+    summary: Updated sass map instructions to reflect new pattern.
+    summaryAdditional: Users are now instructed to define only the variable they wish to change instead of having to define the entire map
+    affectsAccessibility:
```

I remember that at some point in the Sass maps discussion you mentioned that it could be a breaking change for any user that was relying on Sass to clear out any undeclared defaults. Should we include that it might be a breaking change here? 

_Note: Same for utilities changelog_
```suggestion
    isBreaking: true
    affectsAccessibility:
```

---

Good catch! I think it's better safe than sorry here!

Resolved in [e6b26d8](https://github.com/uswds/uswds-site/pull/2077/commits/e6b26d84f667f9e2ab669bd51dbb70d345bc6b64)